<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCRecorderOne</class>
 <widget class="QWidget" name="VSCRecorderOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Master Hard Disk Selecting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../main/vscmainwindows.qrc">
    <normaloff>:/logo/resources/vscsmall.jpg</normaloff>:/logo/resources/vscsmall.jpg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditIP"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="VSCRecorderPanel" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../main/vscmainwindows.qrc">
       <normaloff>:/action/resources/computer.png</normaloff>:/action/resources/computer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSCRecorderPanel</class>
   <extends>QPushButton</extends>
   <header>vscrecorderpanel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main/vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
